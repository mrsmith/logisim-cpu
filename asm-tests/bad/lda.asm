# no addr
lda

# not a number
lda xxx

# too large
lda 256

# too small
lda -1
