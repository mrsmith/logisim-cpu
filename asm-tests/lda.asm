lda 3
hlt
