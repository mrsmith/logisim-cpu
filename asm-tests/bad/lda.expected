lda.asm:2: warn: bad lda instruction `lda`
lda.asm:5: warn: bad lda instruction `lda xxx`
lda.asm:8: warn: bad lda addr `256`
lda.asm:11: warn: bad lda addr `-1`
